### PR TITLE
Chore/DAEF-94 Run MobX in strict mode

### PR DESCRIPTION
**Pull request by darko-mijic**
This PR: 
- re-enables MobX strict mode. 
- updates key file used to import default wallet in acceptance tests to be compatible with testnet 0.3

Note: MobX `autorun` can not be decorated as MobX `action` but code that mutates the state needs to be.

### CODE REVIEW

File: app/stores/WalletsStore.js [hunk 1]
```diff
@@ -218,23 +218,20 @@ export default class WalletsStore extends Store {
...
+  @action _setIsWalletDialogOpen = (isOpen: boolean) => {
+    this.isAddWalletDialogOpen = isOpen;
+  };
+
   goToWalletRoute(walletId: string) {
     const route = this.getWalletRoute(walletId);
     this.actions.router.goToRoute({ route });
   }
 
   _openAddWalletIfNoWallets = () => {
```
Comment (DominikGuzei) [inline app/stores/WalletsStore.js]: I would rename this to `_openAddWalletDialogWhenThereAreNoWallets` to make it more explicit

File: app/stores/WalletsStore.js [hunk 2]
```diff
@@ -216,19 +218,20 @@ export default class WalletsStore extends Store {
...
-    // TODO: investigate why hasLoadedWallets is needed here, it is in hasAnyWallets
-    if (this.hasLoadedWallets) {
-      if (!this.hasAnyWallets) {
-        this.isAddWalletDialogOpen = true;
-      } else {
-        this.isAddWalletDialogOpen = false;
-      }
+    if (this.hasAnyWallets) {
```
Comment (nikolaglumac) [inline app/stores/WalletsStore.js]: I would replace this `if/else` block with the following code:
```
const isOpenWhenThereAreNoWallets = !this.hasAnyWallets;
this._setIsWalletDialogOpen(isOpenWhenThereAreNoWallets);
```

**Comment by DominikGuzei** [comment]
@nikolaglumac you can review again, i did push some updates to address your suggestion!

**Comment by DominikGuzei** [comment]
@nikolaglumac please try running the tests again now, the issue should be fixed (hopefully)

**Comment by nikolaglumac** [comment]
@DominikGuzei all tests have passed with flying colours! Awesome work :)

**Comment by DominikGuzei** [comment]
@nikolaglumac now you have the honor to make your first merge! 😉

**Comment by darko-mijic** [comment]
When CI allows it. @domenkozar ;)

**Comment by domenkozar** [comment]
Waiting on https://github.com/input-output-hk/daedalus/pull/172, once that's merged we can rebase this PR.

**Comment by domenkozar** [comment]
`master` is green again.

**Comment by DominikGuzei** [comment]
thanks @nikolaglumac looks great! 👍

**Comment by nikolaglumac** [comment]
Thanks @DominikGuzei :) Will merge as soon as checks are complete ;)

**Pull request merged**